### DISCUSSION TITLE
When you change a select field value, you need a warning

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#2973] [Feature Requests]
This verges on a bug.  Basically, any select/checkbox/radio, if you edit it and change or delete an existing value, then any old entries that have that value selected can't be edited.  You're going to get a 'Invalid Selection' warning in validation, because validation checks what's set against what's possible and it's going to fail.

Would be nice to get a warning if you make a change and better yet if there was some way to update the old values to the new.  But it definitely jams folks up, and they don't know why they're getting the validation warning.

**Comment by intoeetive** [comment]
Thinking through it, we'd need to perform the check during the validation request. Then have the javascript perform checks whether we should display alert when save is triggered. And then if necessary display alert similar to this https://github.com/ExpressionEngine/ExpressionEngine/pull/2378

**Comment by intoeetive** [comment]
After having experienced this myself, I tend to say this is a bug.

Once you change field settings, you have no way to edit the entries, if they have old values still selected.

The proposed solution is to show the selected/saved values with the entry, even if they are not in list of available options anymore

https://github.com/ExpressionEngine/ExpressionEngine/pull/3801

(also I might need a better name for the PR :) )
